suite/tests/linux/rseq.c

```diff
@@ -424,18 +463,20 @@ test_rseq_branches_once(bool force_restart, int *completions_out, int *restarts_
         "13:\n\t"
         "12:\n\t"
         "5:\n\t"
-        "str xzr, %[rseq_tls]\n\t"
+        "str xzr, %[rseq_cs]\n\t"
         /* clang-format on */
 
-        : [rseq_tls] "=m"(rseq_tls.rseq_cs), [id] "=m"(id),
+        : [rseq_cs] "=m"(reg_rseq->rseq_cs), [id] "=m"(id),
           [completions] "=m"(completions), [restarts] "=m"(restarts),
           [force_restart_write] "=m"(force_restart)
-        : [cpu_id] "m"(rseq_tls.cpu_id), [force_restart] "m"(force_restart)
+        : [cpu_id] "m"(reg_rseq->cpu_id), [force_restart] "m"(force_restart)
         : "x0", "x1", "memory");
 #else
 #    error Unsupported arch
 #endif
     assert(id != RSEQ_CPU_ID_UNINITIALIZED);
+    *completions_out = completions;
```

Comment:
Wait, so this was uninitialized in the caller before?  Or am I misreading this

Reply:
Yes it was missing. Not sure how it worked before.